build/meshmodel_metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"Primary Color":   "#0DA6E5",
+	"Secondary Color": "#0DA6E5",
+	"Shape":           "circle",
+	"Logo URL":        "https://github.com/cncf/artwork/blob/master/projects/linkerd/icon/white/linkerd-icon-white.svg",
```

Comment:
Let's forego inclusion of this property. It's subject to change without our knowledge and only links to one version of the icon. If we were to include a URL, it should be that of the location of the color version as it were to be available in a Meshery Server deployment (e.g. /static/img/meshmodel/components/....)